camayoc/tests/qcs/test_profiles.py

```diff
@@ -9,14 +9,20 @@
 :testtype: functional
 :upstream: yes
 """
+import copy
+import random
 
 import pytest
 
-from uuid import uuid4
-
+from camayoc import api
+from camayoc.utils import uuid4
 from camayoc.qcs_models import HostCredential
 from camayoc.qcs_models import NetworkProfile
-from camayoc.tests.qcs.utils import assert_matches_server
+from camayoc.tests.qcs.utils import (
+    assert_matches_server,
+    assert_profile_update_fails,
+    gen_valid_profile,
+)
 
 CREATE_DATA = ['localhost', '127.0.0.1']
 
```

Comment:
I parametrize some of the tests on these two hosts so we have numeric IP and a string hostname, but because no actual scans are performed, I figured it was an unnecessary complication to pull in data from the config file.

If you disagree, let me know and I can adjust.